lib/routes/eventbrite/events.ts

```diff
@@ -0,0 +1,71 @@
+import type { Route, Data, DataItem } from '@/types'; // Consolidated imports
+import type { Context } from 'hono';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import dayjs from 'dayjs';
+
+export const route: Route = {
+    path: '/:eventType?/:region',
```

Comment:
Place the compulsory parameter before the optional parameter, otherwise the optional parameter becomes compulsory in order to fill in the compulsory parameter